DEVELOP.md

```diff
@@ -0,0 +1,29 @@
+# Resolving issue with gc.h
+
+You will also need a version of the BOEHM GC either provided by the Operating System repositories or compiled from source. Refer to below instructions for your OS respectively.
+
+## Fedora
+
+```
+sudo yum install gc-devel
+```
+
+## Ubuntu
+
+```
+sudo apt-get install libgc-dev
+```
+
+## OSX
+
+```
+git clone git://github.com/ivmai/bdwgc.git
+cd bdwgc
+git clone git://github.com/ivmai/libatomic_ops.git
+autoreconf -vif
+automake --add-missing
+./configure
+make
+make check
+sudo make install
```

Comment:
I think that `brew install bdw-gc` is quite a bit simpler than building manually.